### ISSUE TITLE
`superproductivity` command not found after upgrading to version 10.1.X on Fedora 40

### ISSUE THREAD

**Issue by kadykov** [johannesjo/super-productivity#3664]
### Your Environment

- Version used: 10.1.0 and 10.1.1 (rpm package)
- Operating System and version: Bluefin gts (Fedora Silverblue 40) with Distrobox in BoxBuddy
- Desktop Environment: Gnome 46

### Expected Behavior

When running the command `superproductivity` in the terminal, the superproductivity application should launch successfully.

### Current Behavior

When running the command `superproductivity` in the terminal, the following error message is displayed: `bash: /usr/bin/superproductivity: No such file or directory`

### Steps to Reproduce (for bugs)

1. Install superproductivity as an rpm package in Distrobox on Fedora 40.
2. Open a terminal. 
3. Run the command `superproductivity`.
4. Observe the error message `bash: /usr/bin/superproductivity: No such file or directory`.

### Can you reproduce this reliably?

Yes, this issue can be reliably reproduced every time the command `superproductivity` is run.

### Console Output

```
📦[aleksandr@fedora-40 ~]$ sudo dnf install Downloads/superProductivity-10.0.11.x86_64.rpm 
Last metadata expiration check: 0:02:42 ago on jeu 07 nov 2024 09:12:41.
Dependencies resolved.
================================================================================
 Package                 Architecture Version          Repository          Size
================================================================================
Downgrading:
 superProductivity       x86_64       10.0.11-1        @commandline        79 M

Transaction Summary
================================================================================
Downgrade  1 Package

Total size: 79 M
Is this ok [y/N]: y
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Downgrading      : superProductivity-10.0.11-1.x86_64                     1/2 
  Running scriptlet: superProductivity-10.0.11-1.x86_64                     1/2 
update-alternatives is /sbin/update-alternatives

  Cleanup          : superProductivity-10.1.1-1.x86_64                      2/2 
  Running scriptlet: superProductivity-10.1.1-1.x86_64                      2/2 
/usr/bin/superproductivity has not been configured as an alternative for superproductivity
warning: %postun(superProductivity-10.1.1-1.x86_64) scriptlet failed, exit status 2

Error in POSTUN scriptlet in rpm package superProductivity

Downgraded:
  superProductivity-10.0.11-1.x86_64                                            

Complete!
📦[aleksandr@fedora-40 ~]$ superproductivity
[7666:1107/091557.575330:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
Start app...
09:15:57.730 › { enabled: null, showDevTools: false, mode: 'bottom' }
[7666:1107/091558.124012:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[7666:1107/091558.124087:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
Saving backups to /var/home/aleksandr/.config/superProductivity/backups
09:15:58.226 › Saving backups to /var/home/aleksandr/.config/superProductivity/backups
[7666:1107/091558.231026:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[7666:1107/091558.231094:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[7666:1107/091558.231134:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
09:15:59.078 › No custom styles detected at /var/home/aleksandr/.config/superProductivity/styles.css
[7745:1107/091602.338826:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[7745:1107/091604.552458:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
09:16:06.295 › close, isQuiting: null
09:16:06.298 › Actions to wait for  [ 'SYNC_BEFORE_CLOSE_ID', 'FINISH_DAY_BEFORE_CLOSE_EFFECT' ]
[7666:1107/091608.084545:ERROR:browser_main_loop.cc(283)] GLib-GObject: ../gobject/gsignal.c:2685: instance '0x111c013b7380' has no handler with id '3256'
09:16:08.087 › BEFORE_CLOSE_DONE FINISH_DAY_BEFORE_CLOSE_EFFECT [ 'SYNC_BEFORE_CLOSE_ID' ]
[7745:1107/091608.149589:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
09:16:08.736 › BEFORE_CLOSE_DONE SYNC_BEFORE_CLOSE_ID []
09:16:08.737 › close, isQuiting: null
09:16:08.744 › close, isQuiting: true
09:16:08.775 › Quit after all windows being closed
📦[aleksandr@fedora-40 ~]$ sudo dnf install Downloads/superProductivity-x86_64.rpm 
Last metadata expiration check: 0:03:38 ago on jeu 07 nov 2024 09:12:41.
Dependencies resolved.
================================================================================
 Package                 Architecture Version          Repository          Size
================================================================================
Upgrading:
 superProductivity       x86_64       10.1.1-1         @commandline        79 M

Transaction Summary
================================================================================
Upgrade  1 Package

Total size: 79 M
Is this ok [y/N]: y
Downloading Packages:
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Upgrading        : superProductivity-10.1.1-1.x86_64                      1/2 
  Running scriptlet: superProductivity-10.1.1-1.x86_64                      1/2 
update-alternatives is /sbin/update-alternatives

  Cleanup          : superProductivity-10.0.11-1.x86_64                     2/2 
  Running scriptlet: superProductivity-10.0.11-1.x86_64                     2/2 
/usr/bin/superproductivity has not been configured as an alternative for superproductivity
warning: %postun(superProductivity-10.0.11-1.x86_64) scriptlet failed, exit status 2

Error in POSTUN scriptlet in rpm package superProductivity

Upgraded:
  superProductivity-10.1.1-1.x86_64                                             

Complete!
📦[aleksandr@fedora-40 ~]$ superproductivity
bash: /usr/bin/superproductivity: No such file or directory
📦[aleksandr@fedora-40 ~]$
```

**Comment by johannesjo** [comment]
Not using rpm, but maybe it is "super-productivity" or "superProductivity" or some other variant?

**Comment by kadykov** [comment]
I've also tried these. None of them work. As you can see in the log, it was available in versions 10.0.X as `superproductivity`, but this doesn't work anymore.

**Comment by RealVishy** [comment]
> I've also tried these. None of them work. As you can see in the log, it was available in versions 10.0.X as `superproductivity`, but this doesn't work anymore.

Have you tried super_productivity? The name changed for me as well for the appimage

**Comment by kadykov** [comment]
@RealVishy, thank you for your feedback. I was also able to run `superproductivity`, but only by creating a new container.

It appears that there is a version incompatibility issue, which requires creating a new container when switching between 10.0.X and 10.1.X versions, or vice versa.
For some reason, upgrading or downgrading causes the `superproductivity` command to become unavailable.

A workaround is to create a new container for the 10.1.X version instead of updating the existing one.
I think we can consider this issue resolved.

**Comment by kadykov** [comment]
This issue appears to be resolved in the latest [10.2.3](https://github.com/johannesjo/super-productivity/releases/tag/v10.2.3) release. If you encounter the problem again, please feel free to reopen this issue. Otherwise, I'll go ahead and close it.
